gromacs/__init__.py

```diff
@@ -242,6 +242,17 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+# Try to load environment variables setted by GMXRC
+gmxrc = config.cfg.get("Gromacs", "GMXRC")
+if gmxrc != "":
```

Comment:
Just make this
```
if not gmxrc:
```

Generally better to just test for ``False`` because then one can also set it to ``None`` or ``False``.